core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -461,81 +459,81 @@ public Path getWorkingDirectory() {
   /**
    * {@inheritDoc}
    *
-   * Sets up a lazy connection to Alluxio through mFileSystem. This method will override and
-   * invalidate the current contexts. This must be called before client operations in order to
-   * guarantee the integrity of the contexts, meaning users should not alternate between using the
-   * Hadoop compatible API and native Alluxio API in the same process.
+   * Sets up a lazy connection to Alluxio through mFileSystem and mFsContext. This must be
+   * called before client operations in order to guarantee the integrity of the
+   * {@link FileSystemContext}.
    *
-   * If hadoop file system cache is enabled, this method should only be called when switching user.
+   * If hadoop file system cache is enabled, this method should only be called when switching
+   * user. If it is called twice on the same object an exception will be thrown.
    */
   @SuppressFBWarnings("ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD")
```

Comment:
can this suppression be removed now?

Reply:
I'll see if findbugs still produces a warning